Documentation for global z, cognitive status, and diagnostic variable…

zenourn opened the pull request:
…s in neuropsych export. Also include diagnostic variables in concise output.

zenourn committed:
Documentation for global z, cognitive status, and diagnostic variables in neuropsych export. Also include diagnostic variables in concise output.

zenourn committed:
Minor typo fixed

m-macaskill reviewed (commented):
Oops, it was my understanding that global z was (or should be) the mean of the domain mean z-scores, rather than a grand mean of all constituent scores (which will give more weight to a domain with more tests)? What is the actual calculation, and does this cause issues for our previous publications?

@kylahorne

kylahorne commented:
Global Z is historical and should remain for consistency the mean of the
domain mean z-scores. At a previous PD group meeting this was discussed and
it was decided to produce both scores but I can't remember what the grand
mean was to be called

On Fri, Jan 14, 2022 at 6:44 PM Michael MacAskill ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
>
> Oops, it was my understanding that global z was (or should be) the mean of
> the domain mean z-scores, rather than a grand mean of all constituent
> scores (which will give more weight to a domain with more tests)? What is
> the actual calculation, and does this cause issues for our previous
> publications?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nzbri/chchpd/pull/17#pullrequestreview-852501694>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMKUOP4NKGRW7DF64M3BTNLUV6Z4DANCNFSM5L5BBJZA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kyla-Louise Horne | PhD, BA (Hons), BSc
Research Fellow | Clinical Research Assistant

New Zealand Brain Research Institute, 66 Stewart Street, Christchurch |
Ōtautahi
New Zealand | Aotearoa

Email*/Īmera* ***@***.***
Phone 03 928 1497
ORCID https://orcid.org/0000-0001-6634-8622
Website*/Pae tukutuku* www.nzbri.org/People/horne/

Office Hours: Monday- Friday 8:30am - 6pm

zenourn committed:
Update description of global_z_no_language

zenourn commented:
This came up with Kyla's paper where we described it correctly on pg 393: "... global cognitive score (expressed as an aggregate z score derived from averaging performance from measures conducted across the five cognitive domains)..." We decided to have global_z_historical (which Michael renames to "global_z_no_language" in chchpd) which is the mean of domain means without language, and global_z as the grand mean of all z scores. We discussed this in detail at the time, but effectively taking the mean of all the scores has better properties than when taking it by domain and including language which we now want to do, primarily due to many older sessions only having a single language test thus that having too much weight when taking the mean by "domain". John has also changed which domain several tests are in since the start of the study (was rather fluent with them during the optimal tests paper process) and taking this approach means their weighting is invariant whatever domain they are put in.  I have also done a thorough review of the historical z score code calculation, and did find a small error in relation to what was happening when there are two non-independent tests (i.e., the CVLT short and long). This is now fixed and I really need to get unit tests in place as there is so much complexity in the pipeline where things can go wrong.

zenourn commented:
I've done a quick literature review of how other people calculate global z. There is a real mixture, however the mean of domain means seems to be the most common, followed my the grand mean, and then there are other cases where calculate means by domain but then standardise these before calculating the global z. If we are going to make a change, now is a good time to do it while documenting and because everyone needs to update their data anyway (due to the duplicate data fixes, changes to NA3 imputations when entire test is missing, and fixes to z-score calculations). Do we want to want to make the default global_z the mean of domain means excluding language? Then have a new global_z_grand_mean and global_z_with_language?

m-macaskill commented:
>Do we want to want to make the default global_z the mean of domain means excluding language? Then have a new global_z_grand_mean and global_z_with_language?

I think that would be the ideal outcome - everyone gets to choose the version they want, but the default `global_z` matches what most of us think it was/should have been.

kylahorne commented:
Sounds like a great solution.

On Wed, Jan 19, 2022 at 12:31 PM Michael MacAskill ***@***.***>
wrote:

> Do we want to want to make the default global_z the mean of domain means
> excluding language? Then have a new global_z_grand_mean and
> global_z_with_language?
>
> I think that would be the ideal outcome - everyone gets to choose the
> version they want, but the default global_z matches what most of us think
> it was/should have been.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nzbri/chchpd/pull/17#issuecomment-1015928149>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMKUOPYKDORAGNIMB5U5HQTUWXZ3TANCNFSM5L5BBJZA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Kyla-Louise Horne | PhD, BA (Hons), BSc
Research Fellow | Clinical Research Assistant

New Zealand Brain Research Institute, 66 Stewart Street, Christchurch |
Ōtautahi
New Zealand | Aotearoa

Email*/Īmera* ***@***.***
Phone 03 928 1497
ORCID https://orcid.org/0000-0001-6634-8622
Website*/Pae tukutuku* www.nzbri.org/People/horne/

Office Hours: Monday- Friday 8:30am - 6pm

zenourn committed:
Update global z default and variations

zenourn committed:
Latest agreement about global_z versions

Hopefully getting close to final definitions and is what I'll change redcap export code to return. Not ready for merging - needs to be linked to the new Google Sheets in the new drive once they exist.
